testem.js

```diff
@@ -1,13 +1,24 @@
-/*jshint node:true*/
 module.exports = {
-  "framework": "qunit",
-  "test_page": "tests/index.html?hidepassed",
-  "disable_watching": true,
-  "launch_in_ci": [
-    "PhantomJS"
+  test_page: 'tests/index.html?hidepassed',
+  disable_watching: true,
+  launch_in_ci: [
+    'Chrome'
   ],
-  "launch_in_dev": [
-    "PhantomJS",
-    "Chrome"
-  ]
+  launch_in_dev: [
+    'Chrome'
+  ],
+  browser_args: {
```

Comment:
This seems like added stuff. Are these definitely the right settings? (I don't really know)

Reply:
These are the defaults for new ember apps created when `ember init`. And yup they work on Travis.